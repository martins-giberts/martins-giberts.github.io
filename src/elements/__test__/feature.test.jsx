/* globals describe, it, expect */

import React from 'react'
import {shallow, configure} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import toJson from 'enzyme-to-json'
import Component from '../feature/index'

configure({ adapter: new Adapter() });

describe('elements/Feature test', () => {
  it('should render defaults without throwing an error', () => {
    const wrapper = shallow(<Component name={'Test Feature'} link={'#'} text={'Test text'} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
