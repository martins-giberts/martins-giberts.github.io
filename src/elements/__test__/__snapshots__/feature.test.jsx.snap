// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`elements/Feature test should render defaults without throwing an error 1`] = `
<styled.div>
  <styled.div>
    <styled.a
      href="#"
    >
      <i
        aria-hidden="true"
        className="fa fa-Test Feature"
      />
    </styled.a>
  </styled.div>
  <styled.div
    dangerouslySetInnerHTML={
      Object {
        "__html": "Test text",
      }
    }
  />
  <styled.div>
    <styled.a
      href="#"
    >
      Open
    </styled.a>
  </styled.div>
</styled.div>
`;
